--- core/lib/Drupal/Component/Utility/Xss.php.orig
+++ core/lib/Drupal/Component/Utility/Xss.php
@@ -211,7 +211,7 @@
           // Attribute name, href for instance.
           if (preg_match('/^([-a-zA-Z][-a-zA-Z0-9]*)/', $attributes, $match)) {
             $attribute_name = strtolower($match[1]);
-            $skip = ($attribute_name == 'style' || substr($attribute_name, 0, 2) == 'on');
+            $skip = (substr($attribute_name, 0, 2) == 'on');

             // Values for attributes of type URI should be filtered for
             // potentially malicious protocols (for example, an href-attribute
@@ -226,6 +226,7 @@
               'alt',
               'rel',
               'property',
+              'style',
             ]);

             $working = $mode = 1;
